src/hotspot/share/gc/shared/gcHeapSummary.hpp

```diff
@@ -188,11 +190,12 @@ class G1EvacSummary : public StackObj {
   size_t _failure_waste;
 public:
   G1EvacSummary(size_t allocated, size_t wasted, size_t undo_wasted, size_t unused,
-    size_t used, size_t region_end_waste, uint regions_filled, size_t direct_allocated,
-    size_t failure_used, size_t failure_waste) :
+    size_t used, size_t region_end_waste, uint regions_filled, size_t num_plab_filled,
+    size_t direct_allocated, size_t num_direct_allocated, size_t failure_used, size_t failure_waste) :
```

Comment:
[pre-existing] Shouldn't these parameters be indented to the parameter list start?  As it is, it's hard to spot where the parameters end and the initializers start.